app/assets/javascripts/discourse/tests/integration/components/group-list-setting-test.gjs

```diff
@@ -24,24 +24,21 @@ module("Integration | Component | group-list site-setting", function (hooks) {
 
     this.set(
       "setting",
-      EmberObject.create({
-        allowsNone: undefined,
```

Comment:
The properties removed here are calculated in getters in the proper `SiteSetting` model.